drivers/spi/spi_mcux_lpspi.c

```diff
@@ -550,7 +553,9 @@ static int transceive_dma(const struct device *dev, const struct spi_config *spi
 
 	return ret;
 }
-#endif
+#else
+#define lpspi_inst_has_dma(arg) arg != arg
```

Comment:
Is the `#else`  required? This function is only needed when `CONFIG_SPI_MCUX_LPSPI_DMA` is defined.

Reply:
yes otherwise there will be a build error